code/modules/food/cooking_machines.dm

```diff
@@ -136,6 +136,9 @@ var/global/ingredientLimit = 10
 	else . = ..()
 
 /obj/machinery/cooking/attackby(obj/item/I,mob/user)
+	if(stat & (NOPOWER | BROKEN))
+		to_chat(user, "<span class='warning'> The powers off, it's no good. </span>")
```

Comment:
Nah I don't think it would.

What IS unacceptable is the lack of an apostrophe.